x-pack/plugins/observability/public/hooks/slo/use_fetch_slo_list.ts

```diff
@@ -33,14 +35,20 @@ export interface UseFetchSloListResponse {
   ) => Promise<QueryObserverResult<FindSLOResponse | undefined, unknown>>;
 }
 
+const SHORT_REFETCH_INTERVAL = 5000; // 5 seconds
```

Comment:
The hook now longer takes a `refetchInterval` but a `shouldRefetch` option.

When it is set to true, the hook fetches the SLO List, then depending on if all of the SLOs in the result have data, updates the refetch interval to 1 minute.